.gitignore

```diff
@@ -0,0 +1,5 @@
+*.*~
+**/*.*~
+*~
+**/*~
```

Comment:
We should only need `*~`, according to https://stackoverflow.com/a/4824199/1187277, and this example from https://git-scm.com/docs/gitignore 

>```
>     # ignore objects and archives, anywhere in the tree.
 >     *.[oa]
>```

I'll test this, and make a little addition if I can confirm.